pkg/security/secl/model/accessors_linux.go

```diff
@@ -4,8 +4,8 @@
 // Copyright 2022-present Datadog, Inc.
 // Code generated - DO NOT EDIT.
 
-//go:build unix
```

Comment:
the fact that this was unix allowed this to build successfully on macOS as well, i.e. making work the dd-go rule checker local tests for dev there. Why do we need to change this ?

Reply:
thanks Paul, reverted, back to unix tag

Reply:
thanks !